jetty-server/src/test/java/org/eclipse/jetty/server/HttpConnectionTest.java

```diff
@@ -500,43 +500,42 @@ public void testOKPathDotDotPath() throws Exception
     public void testBadPathDotDotPath() throws Exception
     {
         String response = connector.getResponse("GET /ooops/../../path HTTP/1.0\r\nHost: localhost:80\r\n\n");
-        checkContains(response, 0, "HTTP/1.1 400 Bad URI");
+        checkContains(response, 0, "HTTP/1.1 400 ");
     }
 
     @Test
     public void testOKPathEncodedDotDotPath() throws Exception
     {
         String response = connector.getResponse("GET /ooops/%2e%2e/path HTTP/1.0\r\nHost: localhost:80\r\n\n");
-        checkContains(response, 0, "HTTP/1.1 200 OK");
-        checkContains(response, 0, "pathInfo=/path");
+        checkContains(response, 0, "HTTP/1.1 400 ");
```

Comment:
Change the method name, this test is now "Bad".

So this is actually a behavioral change that may break people out there. Is it unavoidable? Perhaps use a default in `HttpCompliance` that permits it?

Reply:
It is a change in behaviour, but one that we want to do.  Users can get the old behaviour with the compliance mode.
I'm removing these tests from here, since they no longer test what they were, as a %2e%2e can't navigate at all, let alone above the URI root.   The encoding itself is well tested elsewhere.